### PR TITLE
Redesigned and added New Logo

### PR DESCRIPTION
**Pull request by abhinav2712**
I redesigned and added better logo which mixes with background,
add this and consider my PR please, thank you

Here's how it looks now:
image

### CODE REVIEW

**Review by KendallDoesCoding — CHANGES_REQUESTED** [review]
From the image you shared in the comment it looks amazing, but the image does load on the website.

image


**PS: Can I use the new logo for my GitHub profile too?**

File: index.html [hunk 1]
```diff
@@ -41,19 +41,24 @@
...
+   
    <body>
       <!-- Google Tag Manager (noscript) -->
       <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-TR6RS2G"
       height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
       <!-- End Google Tag Manager (noscript) -->
       <section class="hero">
          <nav>
-            <img class="logo" id="logo" src="https://i.ibb.co/zJG8hYw/gsfyqoyk-0.png" alt="gsfyqoyk-0" border="0"></a>
+            <img class="logo" id="logo" src="/portfolio/images/newlogo.png" alt="gsfyqoyk-0" border="0"></a>
```
Comment (KendallDoesCoding) [inline index.html:54]:
```suggestion
            <img class="logo" id="logo" src="/images/newlogo.png" alt="KendallDoesCoding-logo" border="0"></a>

```

i think this should fix the issue?

Comment (abhinav2712) [reply index.html:54]: Let me again give it a shot...

Comment (abhinav2712) [reply index.html:54]: @KendallDoesCoding  yes that will work,

Comment (KendallDoesCoding) [reply index.html:54]: Please commit suggestion.

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
@abhinav2712 Thanks for your contribution!

**Pull request merged**